Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bitcoin_hashes"
-version = "0.8.0"
+version = "0.9.0"
 authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>"]
 license = "CC0-1.0"
 description = "Hash functions used by rust-bitcoin which support rustc 1.14.0"
```

Comment:
Post merge comment. This description is not consistent.

Reply:
Heh, thanks, fixed in #86